### PR TITLE
Lint: fix all govet issue

### PR DESCRIPTION
**Pull request by daixiang0**
Signed-off-by: Xiang Dai <long0dai@foxmail.com>

### CODE REVIEW

File: pkg/scheduler/cache/event_handlers.go [hunk 1]
```diff
@@ -406,7 +406,7 @@ func (sc *SchedulerCache) AddPodGroupV1beta1(obj interface{}) {
 		return
 	}
 
-	pg := &schedulingapi.PodGroup{podgroup, schedulingapi.PodGroupVersionV1Beta1}
+	pg := &schedulingapi.PodGroup{PodGroup: podgroup, Version: schedulingapi.PodGroupVersionV1Beta1}
```
Comment (hzxuzhonghu) [inline pkg/scheduler/cache/event_handlers.go:409]: Is this a warning reported by go vet

Comment (daixiang0) [reply pkg/scheduler/cache/event_handlers.go:409]: Yep, govet need to add key here.

**Pull request merged**